Mathlib/Algebra/Star/SelfAdjoint.lean

```diff
@@ -586,10 +586,14 @@ instance isStarNormal_one [MulOneClass R] [StarMul R] : IsStarNormal (1 : R) :=
   ⟨by simp only [Commute.refl, star_comm_self, star_one]⟩
 #align is_star_normal_one isStarNormal_one
 
-instance isStarNormal_star_self [Mul R] [StarMul R] {x : R} [IsStarNormal x] :
+instance isStarNormal_star [Mul R] [StarMul R] {x : R} [IsStarNormal x] :
```

Comment:
optional. Dot notation would work anytime we have the instance named, but probably this will never come up enough to matter.
```suggestion
protected instance IsStarNormal.star [Mul R] [StarMul R] {x : R} [IsStarNormal x] :
```